src/parameterizations/vertical/MOM_energetic_PBL.F90

```diff
@@ -829,6 +841,8 @@ subroutine ePBL_column(h, u, v, T0, S0, dSV_dT, dSV_dS, TKE_forcing, B_flux, abs
   max_MLD = 0.0 ; do k=1,nz ; max_MLD = max_MLD + h(k)*GV%H_to_Z ; enddo
   !min_MLD will initialize as 0.
   min_MLD = 0.0
+  ! Set values of the wrong signs to indicate that these changes are not based on valid estimates
+    ! dMLD_min = -1.0*US%m_to_Z ; dMLD_max = 1.0*US%m_to_Z
```

Comment:
Did you mean to comment this line out?

Reply:
Good catch, and no I did not intend for this line to be commented out. However, because the code only starts using these after the 3rd iteration, and I am pretty sure that both of these are set in the first two iterations, the answers should be OK, but I will fix this anyway.